### ISSUE TITLE
Command-line should have a 't' command for bidirectional transfer

### ISSUE THREAD

**Issue by jamesbowman** [jamesbowman/spidriver#8]
't' should act like 'w', but print out the received bytes.

Suggested in this thread:
http://gameduino2.proboards.com/thread/852/spidriver-running-gameduino3?page=1&scrollTo=2221

**Comment by maxnil** [comment]
Any progress on this?

**Comment by 0xStuart** [comment]
Is that why I cant seem to get anything out of the Atmel M90E26?  You have to write 8 bits, and it immediately returns 16 bits.  Except I see nothing.

This is from the manual.

![image](https://user-images.githubusercontent.com/432831/125203608-06e84800-e271-11eb-8872-e8dcd80b37d1.png)

On the spidriver, I see the clock being interrupted, between flipping between write and read.

**Comment by jamesbowman** [comment]
This diagram looks half-duplex, so the right sequence should be:

    spicl <device> s w 0x40 r 2 u

**Comment by 0xStuart** [comment]
It operates like it's half-duplex, but the manual says full-duplex.
The SCK doesn't look the same as the diagram, don't know if that's relevant.

![image](https://user-images.githubusercontent.com/432831/125206116-65b3be80-e27d-11eb-9b89-9ee56e259994.png)

**Comment by jamesbowman** [comment]
This looks fine - the gaps on SCK are just the inter-byte pauses.

![image](https://user-images.githubusercontent.com/894763/125206343-a1c62f80-e23b-11eb-9ded-b5bd0edc8b0d.png)

Hmm, one idea... datasheet does say that it expects a '1' bit before the 7-bit register address. So changing 0x49 there to 0xc9 might work?

**Comment by 0xStuart** [comment]
Good call. That wasn't quite it, but I got it working.  I discovered I have a different chip variation to the manual.  Thanks.

**Comment by benner** [comment]
I have same issue - I need to receive same time I'm transmitting (full duplex).
